ProjectTemplates/templates/Worker-CSharp/Company.Application1.csproj

```diff
@@ -13,7 +13,9 @@
 
   <ItemGroup>
     <PackageReference Include="Grpc.AspNetCore" Version="2.32.0" />
-    <!--<PackageReference Include="Microsoft.AspNetCore.Authentication.JwtBearer" Version="5.0.0-0" />-->
     <PackageReference Include="Microsoft.Identity.Web" Version="1.*" />
+    <!--#if (GenerateGraph) -->
```

Comment:
Huh, I didn't know about this trick. How does it work?

Reply:
the dotnet new tool will do the template extensions based on the command line options and the variables computed there. `GenerateGraph` is one of those variables, it's defined [here](https://github.com/AzureAD/microsoft-identity-web/pull/901/files#diff-70d6aa29757c7ed2d2380a8ba03f394e95b5585f3e4604b48c74625b592147e3R191)